weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/customWeaveType.types.ts

```diff
@@ -38,9 +38,11 @@ export const isCustomWeaveTypePayload = (
     if (data.load_op == null) {
       return false;
     }
+    // Load op can be a ref, so we need to unwrap it.
+    const load_op = data.load_op?.__val__ ?? data.load_op;
```

Comment:
Does this need to be `?.__val__` - at this point since `data.load_op != null`?

Reply:
This line of code is also a little confusing because below it seems like we could still be in the case that `data.load_op` is a string.

Reply:
Good call, the correct fix here was to ensure that we resolve those `__ref__/__val__` patterns in the `mapCallsToDatasetRows` function in the case where an op has one output and its a custom weave type. I'm removing this change.